ui/src/shared/utils/analytics.ts

```diff
@@ -23,3 +25,25 @@ export const fireUserDataReady = function fireUserDataReady(
     },
   })
 }
+
+export const fireGAEvent = (event: string, payload: object = {}) => {
```

Comment:
```suggestion
export const fireEvent = (event: string, payload: object = {}) => {
```

small thing, but I think dropping mention of GA is probably ideal - we abstract the underlying mechanism and only worry about passing in a string and a payload. we turn this function into a separate thing that handles events, regardless of the underlying mechanism.

Reply:
had a thought: what do you think of moving this file under `ui/cloud` since events are only really fired in a cloud context?

Reply:
I am assuming all of this will be refactored once the actual implementation of GA eventing goes out.